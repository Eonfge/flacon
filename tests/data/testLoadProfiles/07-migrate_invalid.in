[Aac]
Bitrate=128
Quality=112
UseQuality=false

[Flac]
Compression=1
ReplayGain=Album

[Mp3]
Bitrate=128
Preset=cbrKbps
Quality=1
ReplayGain=Album

[Ogg]
MaxBitrate=@Invalid()
MinBitrate=@Invalid()
NormBitrate=@Invalid()
Quality=2.3
ReplayGain=Track
UseQuality=false

[Opus]
Bitrate=125
BitrateType=VBR

[WV]
Compression=0
ReplayGain=Track
