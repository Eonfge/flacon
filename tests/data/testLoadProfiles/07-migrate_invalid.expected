[Profiles/AAC]
Name=AAC
Format=AAC
Bitrate=128
Quality=112
UseQuality=false

[Profiles/FLAC]
Name=Flac
Format=FLAC
Compression=1
ReplayGain=Album

[Profiles/MP3]
Name=MP3
Format=MP3
Bitrate=128
Preset=cbrKbps
Quality=1
ReplayGain=Album

[Profiles/OGG]
Name=OGG
Format=OGG
MaxBitrate=
MinBitrate=
NormBitrate=
Quality=2.3
ReplayGain=Track
UseQuality=false

[Profiles/OPUS]
Name=Opus
Format=OPUS
Bitrate=125
BitrateType=VBR

[Profiles/WV]
Name=WavPack
Format=WV
Compression=0
ReplayGain=Track
